app/jobs/trs/update_professional_status_job.rb

```diff
@@ -12,8 +12,12 @@ def perform(trainee)
 
       UpdateProfessionalStatus.call(trainee:)
 
-      if trainee.recommended_for_award? && survey_should_be_scheduled?
-        Survey::SendJob.set(wait: Settings.qualtrics.days_delayed.days).perform_later(trainee: trainee, event_type: :award)
+      # Handle the award state transition when the trainee is recommended for award
+      # TRS returns 204 No Content, so we need to use our own award date
+      if trainee.recommended_for_award? && trainee.outcome_date.present?
+        trainee.award_qts!(trainee.outcome_date)
```

Comment:
just confirming if todays date is more appropriate here in the slack channel

Reply:
<img width="1333" alt="Screenshot 2025-05-07 at 13 35 20" src="https://github.com/user-attachments/assets/50ffbe3f-66f6-4597-afee-3e96b6413050" />